static/js/publisher/release/actions/releases.js

```diff
@@ -1,5 +1,157 @@
+import {
+  RISKS_WITH_AVAILABLE as RISKS,
+  DEFAULT_ERROR_MESSAGE as ERROR_MESSAGE
+} from "../constants";
+
+import { hideNotification, showNotification } from "./globalNotification";
+import { cancelPendingReleases } from "./pendingReleases";
+import { releaseRevisionSuccess, closeChannelSuccess } from "./channelMap";
+import { updateRevisions } from "./revisions";
+
+import {
+  fetchReleasesHistory,
+  fetchReleases,
+  fetchClose
+} from "../api/releases";
+
+import { getRevisionsMap, initReleasesData } from "../releasesState";
+
 export const UPDATE_RELEASES = "UPDATE_RELEASES";
 
+function updateReleasesData(releasesData) {
+  return dispatch => {
+    // init channel data in revisions list
+    const revisionsMap = getRevisionsMap(releasesData.revisions);
+    initReleasesData(revisionsMap, releasesData.releases);
+    dispatch(updateRevisions(revisionsMap));
+    dispatch(updateReleases(releasesData.releases));
+  };
+}
+
+function handleCloseResponse(dispatch, json, channels) {
+  if (json.success) {
+    if (json.closed_channels && json.closed_channels.length > 0) {
+      json.closed_channels.forEach(channel => {
+        // make sure channels without track name get prefixed with 'latest'
+        if (RISKS.indexOf(channel.split("/")[0]) !== -1) {
+          // TODO: This should be the default track, not latest
```

Comment:
I'm not sure if this comment is true (?) If channel is only a risk (like "beta") is it really "default-track/beta"? Or is it "latest/beta"? ... Default tracks are confusing...

Reply:
it will be `default-track/beta`, though for most people default-track will = latest